docs/src/layouts/doc-layout/use-search.js

```diff
@@ -38,8 +37,7 @@ export default function useSearch (scope, $q, $route) {
   const searchHasFocus = ref(false)
   const searchActiveId = ref(null)
   const searchInputRef = ref(null)
-
-  const $router = useRouter()
+  const focusByKbd = ref(false)
```

Comment:
**Always** avoid abbreviation, use `focusByKeyboard` instead and change the related event name accordingly